jobs/scripts/4.3-tests.sh

```diff
@@ -4,4 +4,4 @@
 bin/buildout -c jenkins.cfg
 
 ROBOT_BROWSER=chrome
-bin/jenkins-alltests -1
+xvfb-run -a --server-args='-screen 0 1920x1200x24' bin/jenkins-alltests -1
```

Comment:
@gforcada doesn't this always bind to screen 0 which leads to collisions?

Reply:
@Rotonen can shed some light on it, I'm a mere copy&paster here :sweat_smile: but so far on Jenkins it did not fail due to that